server/routes/index.ts

```diff
@@ -22,6 +22,7 @@ export default function routes(services: Services): Router {
 
   const prisonerController = new PrisonerController(services.expectedArrivalsService)
   get('/prisoner/:prisonNumber/image', prisonerController.getImage())
+  get('/prisoner/:UUID/confirm-arrival', prisonerController.getPrisoner())
```

Comment:
can we call this `prisonerController.confimArrival` to match the name of the page?

It could probably do with being its own controller.. but happy to leave for now.

We need to keep an eye on this files as its going to get messy really quickly

Reply:
Name changed and moved out into its own controller